Documentation/known-issues.md

```diff
@@ -1,4 +1,4 @@
 Known issues & workarounds
 ==========================
 
-This document now lives in the [known issues document](https://github.com/dotnet/core/blob/master/cli/known-issues.md) in the dotnet/core repository. 
+This document now lives in the  in the dotnet/core repository. 
```

Comment:
If you are going to keep this doc as a separate one (which I'm not in favor of), shouldn't you provide a link to it?